codegen/protocol-test-codegen-local/Tests/swift-codegen/waitersTests/WaitersTests.swift

```diff
@@ -0,0 +1,65 @@
+import XCTest
+@testable import waiters
+@testable import ClientRuntime
+
+class WaitersTests: XCTestCase {
```

Comment:
I would expect these tests to be codegenerated and end up in a location like `/codegen/protocol-test-codegen-local/build/smithyprojections/protocol-test-codegen-local/....` following the pattern as the other protocol tests